test/fixtures/authors.yml

```diff
@@ -24,11 +25,13 @@ two:
   user: yo
   thesis: two
   graduation_confirmed: true
+  proquest_allowed: true
 
 three:
   user: basic
   thesis: two
   graduation_confirmed: false
+  proquest_allowed: false
```

Comment:
Are we also accounting for the `null` value in any tests?

Reply:
We are in the thesis_helper_test coverage for the ProQuest status methods: https://github.com/MITLibraries/thing/blob/6050cf75e7fcce10b7dda28432726f340d25a522/test/helpers/thesis_helper_test.rb#L193-L224

Reply:
Oh neat, I didn't realize that GitHub now translates urls to code snippets!